jared-alrightjones/Homework-2/temp-converter/js/main.js

```diff
@@ -0,0 +1,43 @@
+/*
+    1. Create a function that accepts a temperature value in Fahrenheit and converts it to Celsius.
+      Note: Equation to convert F to C:
+      C = (F - 32) / 1.8
+      Test your function.
+    2. Create a second function that accepts a temperature value in Celsius and converts it to Fahrenheit.
+      Note: Equation to convert C to F:
+      F = 1.8 * C + 32
+      Test your function.
+
+    BONUS 1: Update your functions to use the value entered in the text box on the web page, and to perform calculations only when the appropriate button is clicked:
+      1.  Set the value of the temp variable to the value of the element with the id value 'temperature'.
+          (Hint: read about the document.querySelector() method.)
+      2.  Set the appropriate function to run when each button is clicked.
+        (Hint: read about the addEventListener() method.)
+
+    BONUS 2: Display the result of each calculation in the browser window rather than in the console:
+      1.  At the end of each function, set the text of the element with the id value 'result' to the same text string you were previously logging to the console.
+
+    BONUS 3: Clear the contents of the text box after you've displayed your results, so a user can easily enter another temperature to convert. (Hint: Read about the 'value' property in JavaScript. What string value could you assign as the value of an input box that would give it no content to display?)
+*/
+
+function fahrenheit_to_celsius (temperature, metric) {
+	if (metric === 'fahrenheit') {
+		console.log(temp - 32 / 1.8);
+	} else {
+	   console.log(temp);
+	}
+}
+fahrenheit_to_celsius(76, 'fahrenheit');
+fahrenheit_to_celsius(50, 'fahrenheit');
+fahrenheit_to_celsius(101, 'fahrenheit');
+
+function celsiusToF (temp, metric) {
```

Comment:
I like that you experimented with a couple different ways of naming the function and the parameters. In general, you want to harmonize these within the same app, so it's clear that they're related -- so you might rename the first function `fahrenheitToC` and its first parameter to `temp`. But in general a great job here.